src/components/organisms/FetchList/FetchList.js

```diff
@@ -0,0 +1,31 @@
+import React, { useEffect, useState } from 'react';
+import { List } from '../List/List';
+
+export const FetchList = () => {
+  const [error, setError] = useState(null);
+  const [isLoaded, setIsLoaded] = useState(false);
+  const [hits, setHits] = useState([]);
+
+  useEffect(() => {
+    fetch(`https://itunes.apple.com/us/rss/topalbums/limit=100/json`)
```

Comment:
To nie uwaga, ale ciekawostka: przykład z większych projektów - bardzo często, gdy uderzasz do API, będziesz miała baseUrl (w tym wypadku będzie to "https://itunes.apple.com/us/rss/". 

Takie stałe trzyma się często pod folderem "constants" lub "config". Wtedy, za każdym razem gdy wykonujesz zapytanie po prostu używasz baseUrl + endpoint (w tym wypadku endpoint to "topalbums/limit=100/json". 

Jest to głównie używane żeby zredukować liczbę "zahardcodowanych" stałych + najczęsciej w dużych projektach masz kilka srodowisk które używają różnych baseUrl (np. dev.com/, staging.com/, production.com/). Możesz wtedy bez problemu przełączać się między nimi, bez koniecznosci zmiany url we wszelkich możliwych plikach :)